src/libs/data-utils.js

```diff
@@ -19,8 +20,8 @@ import * as Utils from 'src/libs/utils'
 
 
 const warningBoxStyle = {
-  border: `1px solid ${colors.warning(0.85)}`,
-  backgroundColor: colors.warning(0.4),
+  border: '1px solid #DC8412',
+  backgroundColor: colors.warning(1),
```

Comment:
Nit: Equivalent to `colors.warning()`.

Non-nit: This is now the same as the text color below. Looks like you're using `colors.light(.1)` for the text color of the warning you added. Should be able to put that here instead.